_inc/lib/core-api/wpcom-endpoints/memberships.php

```diff
@@ -14,6 +14,35 @@
  */
 class WPCOM_REST_API_V2_Endpoint_Memberships extends WP_REST_Controller {
 
+	/**
+	 * Currencies should be supported by Stripe.
+	 *
+	 * @link https://stripe.com/docs/currencies
+	 *
+	 * List has to be in sync with the Recurring Payments block in Jetpack and the Memberships library in WP.com.
+	 * @link https://github.com/Automattic/jetpack/blob/2e7e2c4dbe6348933075cef0a8ca75dd21b20657/extensions/blocks/recurring-payments/index.js#L81
+	 * @see Memberships_Product::SUPPORTED_CURRENCIES
+	 */
+	const SUPPORTED_CURRENCIES = array(
```

Comment:
I'd rather cut this, and leave the currency validation to the library. It'll be the same result for the client, and we won't need to worry about the duplicate data.

Reply:
019243b2e03cb53575e7c9fdb7d41bb78eb14c09